docs/website/content/docs/language.md

```diff
@@ -37,6 +37,49 @@ Notes about sysl syntax:
   - All lines after `:` should be indented. The only exception to this rule is when you want to use the shortcut `...`.
   - The `...` (aka shortcut) means that we don't have enough details yet to describe how this endpoint behaves. Sysl allows you to take an iterative approach in documenting the behaviour. You add more as you know more.
 
+### Application Names
```

Comment:
This isn't openAPI?

Reply:
Ahh, yeah some of these docs are related to how to define an application in Sysl so that they can be exported to Openapi3. Happy to move this to another PR if needed